### ISSUE TITLE
[Bug]: ZUI Total Downloads on Image Subpage Not Updating

### ISSUE THREAD

**Issue by bfreidank** [project-zot/zui#352]
**Describe the bug**
Main page list of current images updates Downloads count correctly but navigating to specific image sub page shows Total downloads count as 0.

**To Reproduce**
1. Go to zui for zot server with images that have been accessed.
2. Click on any image in list to navigate to image sub page
3. Scroll down to Total Downloads located in the right of page showing statistics
4. See error showing Total downloads as 0

**Expected behavior**
Total downloads update to current total downloads. Alternatively could remove total downloads to specific tag layers page or next to specific tag.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.2 LTS x86_64
 - Browser: Edge Chromium, Mozilla Firefox Snap for Ubuntu
 - Version: Version 113.0.1774.35 (Official build) (64-bit), 112.0.2 (64-bit)

**Additional context**

**Comment by zainkhanh1892** [comment]
All images besides machine/bootkit/provision-rootfs, show incorrect image downloads from subpages. However, instead of showing 0 downloads, it showed 4. Either a backend/API issue or a frontend UI issue.

**Comment by andaaron** [comment]
Hi @zainkhanh1892, can you please clarify what you mean by incorrect?

After the user first uploads an image it should have 0 downloads. That count should increment by 1 every time the user download that image.
If you have pre-existing images on an older server, which have been downloaded before, it is normal for that count to be non-zero.

**Comment by zainkhanh1892** [comment]
Yeah of course, based on my understanding of the error on the main page it shows all the images in the repo and to the right of it there's downloads. If I'm looking at c3/ubuntu/openj9-amd64 it says downloads to be 25274 on the main page, but if I click on the subpage total downloads are four. It's the same for a lot of images such as c3/debian/openj9-amd64, also show 4 downloads on the subpage where on the main page it shows 25305 downloads. The only image that has a accurate number of downloads is machine/bootkit/provision-rootfs where on the main page and subpage they match to a total of 970 downloads. Either I'm reading the main page downloads wrong or something else. Let me know if that makes more sense!

**Comment by andaaron** [comment]
I see now you are referring to https://zothub.io/home, this site is running an older deployment:

Some notes:
- Both the UI and backend have had changes in the past year, and we had been waiting for the oci dist spec 1.1.0 to be released before we release zot 2.0.0 and update the zothub.io. The build deployed there is from late last year, so there may be bugs which have been fixed in the meantime. The UI in particular has had a lot of changes.
- There are new images pushed on a regular basis on this deployment.

It does seem to be a backend bug as I checked on a recent build and it is still inconsistent. Looking into it with @laurentiuNiculae.
